app.py

```diff
@@ -17,7 +17,72 @@
 
 """This is the main script of the template project."""
 
-from template.version import __version__
+from typing import List, Optional
+
+import click
+import logging
+from github import Github
+from github.GithubException import GithubException
+from thoth.common import OpenShift
+
+from version import __version__, __title__
+
+_LOGGER = logging.getLogger(__title__)
+logging.basicConfig(level=logging.INFO)
+
+
+@click.command()
+@click.option(
+    "--repositories", help="Repositories to be analysed (e.g thoth-station/performance)",
+)
+@click.option(
+    "--organizations",
+    "-o",
+    type=str,
+    required=False,
+    help="Organizations (all of their repositories) to be analysed (e.g. AICoE)",
+)
+def main(
+    repositories: Optional[str], organizations: Optional[str],
+):
+    """Command Line Interface for SrcOpsMetrics."""
+    repositories = "" if repositories is None else repositories
+    organizations = "" if organizations is None else organizations
+
+    gh = Github()
+
+    orgs = organizations.split(",")
+
+    repos = []
+    for org in orgs:
+        try:
+            gh_org = gh.get_organization(org)
+            for repo in gh_org.get_repos():
+                if repo.archived:
+                    _LOGGER.info("repository %s is archived, therefore skipped" % repo.full_name)
+                else:
+                    repos.append(repo.full_name)
+        except GithubException:
+            _LOGGER.error("organization %s was not recognized by GitHub API" % org)
+
+    repos.extend(repositories.split(","))
+
+    schedule_repositories(repositories=repos)
+
+
+def schedule_repositories(repositories: List[str]) -> None:
+    """Schedule workflows for repositories.
+
+    Repositories are also gathered from all of the organizations passed.
+
+    :param organizations:str: List of organizations in string format: org1,org2,org3,...
+    :param repositories:str: List of repositories in string format: repo1,repo2,...
+    """
+    oc = OpenShift()
+    for repo in repositories:
+        oc.schedule_srcopsmetrics_workflow(repository=repo)
+
 
 if __name__ == "__main__":
-    print(f"A template project with Thoth integration, v{__version__}.")
+    print(f"mi-scheduler for scheduling mi workflows v{__version__}.")
```

Comment:
It would be better to use a logger here. Logs are automatically turned into JSON format in cluster deployment and can be turned on/suppressed based on the loglevel used.